rfcs/0145-doc-strings.md

```diff
@@ -140,7 +140,7 @@ A **documentable node** can be:
 
 - Expression 
 - Binding 
-- Lambda Formal
+- Function Argument
```

Comment:
This change makes it even more confusing IMO – I think function argument without further qualification usually refers to “(actual) arguments” – i.e. the expression passed to functions. The names/bindings in the function definition are often called “(formal) parameters” or “formal arguments”. This variant is not specific to Lisp as it is used e.g. in C++ standard. See also https://en.wikipedia.org/wiki/Parameter_(computer_programming)

Reply:
I am happy with "Function parameters". This seems better and is on the correct side of your definition.